lib/dor/text_extraction/abbyy/split_alto.rb

```diff
@@ -13,9 +13,12 @@ def initialize(alto_path:)
         end
 
         def write_files
-          split_to_files.each do |filename, xml_content|
-            File.write(output_path(filename), xml_content)
+          pages = doc.css('//Page')
+          pages.each_with_index do |page, i|
+            filename = "#{File.basename(page_filenames[i], File.extname(page_filenames[i]))}.xml"
+            File.write(output_path(filename), xml_structure(page).to_xml)
           end
+          true
```

Comment:
If you do not return true (or something else) here, the method return result is an array of nokogiri documents, which can get very very large.  We do not care about the result, so just return true